storage/client_raft_test.go

```diff
@@ -274,6 +274,23 @@ func TestFailedReplicaChange(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	// Wait for the range to sync to both replicas (mainly so leaktest doesn't
```

Comment:
Would be a lot nicer if we had some way to more generally wait for the system to quiesce before shutdown. We can't put this code in every test going forward.